packages/node_modules/@webex/webex-core/test/integration/spec/credentials/credentials.js

```diff
@@ -19,6 +19,32 @@ describe('webex-core', () => {
         user = u;
       }));
 
+    describe('#extractOrgIdFromUserToken()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('define webex', () => {
+        webex = new WebexCore({
+          credentials: user.token
+        });
+
+        credentials = webex.credentials;
+      });
+
+      it('should return the OrgId when the provided token is valid', () => {
+        // The refresh token is formatted like a normal user token.
+        const token = user.token.refresh_token;
+
+        assert.equal(credentials.extractOrgIdFromUserToken(token), user.orgId);
+      });
+
+      it('should return undefined when provided an invalid token', () =>
```

Comment:
Same comment as your other PR, since these tests don't integrate with anything, they would be better suited as unit tests.

Reply:
This has been moved.